0MI0800304/7.03/ConsoleApplication1/problem123.cpp

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <fstream>
+//problem 1
+void writeData() {
+	std::ofstream file("result.txt");
+	if (!file.is_open()) {
+		std::cout << "Error!" << std::endl;
+		return;
+	}
+	int a = 0, b = 0, c = 0;
+	std::cin >> a >> b >> c;
+	file << a + b + c << " " << a * b * c << std::endl;
+	file.close();
+
+}
+void readData() {
+	std::ifstream file("result.txt");
+	if (!file.is_open()) {
+		std::cout << "Error!" << std::endl;
+		return;
+	}
+	int a = 0, b = 0;
+	file >> a >> b;
+	std::cout << a - b;
+	file.close();
+	}
+//problem 2
+int charCount() {
+	std::ifstream file("myFile.txt");
+	int count = 0;
+	if (!file.is_open())
+	{
+		std::cout << "Error!" <<std::endl;
+		return -1;
+	}
+	while (!file.eof()){
+		char ch;
+		file.get(ch);
+		if (ch != 'a' && ch != 'd')count++;
+	}
+	file.close();
+	return count;
+}
+//problem 3
+struct Pair {
+	unsigned x;
+	unsigned y;
+};
+Pair createPair(unsigned x,unsigned y) {
+	Pair pair;
+	pair.x = x;
+	pair.y = y;
+}
+void initPair(Pair& toInit, unsigned x, unsigned y) {
+	toInit.x = x;
+	toInit.y = y;
+}
+Pair readPairFromFile(std::ifstream& inFile) {
+	Pair pair;
+	inFile >> pair.x >> pair.y;
+	inFile.close();
+	return pair;
+}
+void writePairToFile(std::ofstream& outFile, const Pair& toWrite) {
+	outFile << toWrite.x << " " << toWrite.y;
+	outFile.close();
+}
+struct Relation {
+	Pair pairArr[25];
+	unsigned size = 0;
+};
+bool addPair(Relation& arg, Pair& toInsert){
+	if (arg.size < 25) {
+		arg.pairArr[arg.size++] = toInsert;
+		return 1;
```

Comment:
What's the deal with using `0` & `1` instead of `true` and `false`. Don't forget - code is written once but is read many times. The more readable you write it - the better :))